soccer/gameplay/evaluation/ball.py

```diff
@@ -6,12 +6,12 @@
 
 def is_moving_towards_our_goal():
     # see if the ball is moving much
-    if main.ball().vel.mag() > 0.3:
+    if main.ball().vel.mag() > 0.1:
         # see if it's moving somewhat towards our goal
         if main.ball().vel.dot(robocup.Point(0, -1)) > 0:
-            ball_path = robocup.Segment(main.ball().pos, (main.ball().pos + main.ball().vel.normalized()))
+            ball_path = robocup.Line(main.ball().pos, (main.ball().pos + main.ball().vel.normalized()))
 
-            fudge_factor = 0.05
+            fudge_factor = constants.Robot.Radius * 2
```

Comment:
Fair enough.  It just seems a little odd that if we had bigger robots, the threshold for a ball coming at our goal would be bigger...